docs/sources/dashboards/share-dashboards-panels/index.md

```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
 
-The dashboard export action creates a Grafana JSON file that contains everything you need, including layout, variables, styles, data sources, queries, and so on, so that you can later import the dashboard.
+### Share a snapshot
 
-1. Click **Dashboards** in the main menu.
-1. Open the dashboard you want to export.
-1. Click **Share** in the top-right corner.
-1. Click **Export**.
+A dashboard snapshot publicly shares a dashboard while removing sensitive data such as queries and panel links, leaving only visible metrics and series names. Anyone with the link can access the snapshot.
+
+<!-- is the following para still true? -->
+
+You can publish snapshots to your local instance or to [snapshots.raintank.io](http://snapshots.raintank.io). The latter is a free service provided by Grafana Labs that enables you to publish dashboard snapshots to an external Grafana instance. Anyone with the link can view it. You can set an expiration time if you want the snapshot removed after a certain time period.
 
-   If you're exporting the dashboard to use in another instance, with different data source UIDs, enable the **Export for sharing externally** switch.
+To share your dashboard with anyone as a snapshot, follow these steps.
 
-1. Click **Save to file**.
+1. Click **Dashboards** in the main menu.
+1. Click the dashboard you want to share.
+1. Click the **Share** drop-down in the top-right corner and select **Share snapshot**.
+1. In the **Snapshot name** field, enter a descriptive title for the snapshot.
+1. Select one of the following expiration options for the snapshot:
+   - **1 Hour**
+   - **1 Day**
+   - **1 Week**
+   - **Never**
+1. Click **Publish snapshot**.
+1. (Optional) If you want to see the other snapshots shared from your organization, click the **View all snapshots** link.
 
-Grafana downloads a JSON file to your local machine.
+   You can also navigate to **Dashboards > Snapshots** in the primary menu.
 
-#### Make a dashboard portable
+1. Click the **X** at the top-right corner to close the share drawer.
 
-If you want to export a dashboard for others to use, you can add template variables for things like a metric prefix (use a constant variable) and server name.
+### Delete a snapshot
 
-A template variable of the type `Constant` is automatically hidden in the dashboard, and is also added as a required input when the dashboard is imported.
+To delete existing snapshots, follow these steps:
 
-## Export dashboard as PDF
+1. Navigate to **Dashboards > Snapshots** in the main menu.
+1. Click the red **x** next to the snapshot that you want to delete.
 
-You can generate and save PDF files of any dashboard.
+The snapshot is immediately deleted. You may need to clear your browser cache or use a private or incognito browser to confirm this.
 
-{{< admonition type="note" >}}
-Available in [Grafana Enterprise](ref:grafana-enterprise) and [Grafana Cloud](/docs/grafana-cloud/).
-{{< /admonition >}}
+### Export a dashboard as JSON
```

Comment:
Will need to include Export dashboard as PDF section too (before JSON since the button will appear first on the export menu)